lstchain/tools/lstchain_create_dl2_file.py

```diff
@@ -0,0 +1,161 @@
+"""
+Create h5 file with reconstruction of energy, disp and gamma/hadron separation of events.
+"""
+
+import joblib
+import numpy as np
+import pandas as pd
+from ctapipe.core import Provenance, Tool, traits
+from lstchain.io import (
+    get_dataset_keys,
+    read_configuration_file,
+    standard_config,
+    write_dl2_dataframe,
+)
+from lstchain.io.io import (
+    dl1_images_lstcam_key,
+    dl1_params_lstcam_key,
+    dl1_params_src_dep_lstcam_key,
+)
+from lstchain.reco import dl1_to_dl2
+from lstchain.reco.utils import filter_events, impute_pointing, add_delta_t_key
+from tables import open_file
+
+
+class ReconstructionHDF5Writer(Tool):
+
+    name = "ReconstructionHDF5Writer"
+    description = "Generate a HDF5 file with reconstructed energy, disp and hadroness of events"
+
+    input = traits.Path(
+        help="Path to a DL1 HDF5 file", directory_ok=False, exists=True
+    ).tag(config=True)
+    path_models = traits.Path(
+        help="Path where to find the trained RF models", file_ok=False
+    ).tag(config=True)
+    output_dir = traits.Path(
+        help="Path where to store the reconstructed DL2", file_ok=False
+    ).tag(config=True)
+    config_file = traits.Path(
+        help="Path to a configuration file. If none is given, a standard configuration is applied",
+        directory_ok=False,
+    ).tag(config=True)
+
+    aliases = {
+        ("i", "input"): "ReconstructionHDF5Writer.input",
+        ("o", "output"): "ReconstructionHDF5Writer.output_dir",
+        "models": "ReconstructionHDF5Writer.path_models",
+        "config": "ReconstructionHDF5Writer.config_file",
```

Comment:
No need for this, `Tool` already comes with `config` traitlet for setting the configuration file